### ISSUE TITLE
Revise the accountRegIdHistory of the Accounts structure

### ISSUE THREAD

**Issue by MilkywayPirate** [Concordium/concordium-node#1096]
In the same spirit as https://github.com/Concordium/concordium-node/issues/1040 we should also move the account registry id -> account index mapping to LMDB. (The account map lmdb database is suitable for this)

This will lower the growth rate of the database, entries in the current map are mostly on disk so it is unlikely it will have much effect on memory usage.

**Comment by abizjak** [comment]
As far as I remember this is not cached, so it is doubtful that removing it will decrease memory usage in normal operation.

**Comment by EmilLa1** [comment]
> As far as I remember this is not cached, so it is doubtful that removing it will decrease memory usage in normal operation.

Yes, you are right. Under normal operation we should not expect reduced memory usage by this initiative. I have updated the description.
